index.html

```diff
@@ -343,67 +285,49 @@ <h3>
           </li>
           <li>Let |promise:Promise| be [=a new promise=].
           </li>
-          <li>Run the following steps <a>in parallel</a>:
+          <li>
+            <a>Queue a global task</a> on the <a>screen wake lock task
```

Comment:
Do we still need to queue a global task here? We're already in the main thread at this point.

Reply:
We don't technically need to but this provides implementations with the freedom to not return a settled Promise synchronously from this method, as otherwise this method doesn't need to be Promise-returning at all.

Reply:
> as otherwise this method doesn't need to be Promise-returning at all.

Wouldn't this be a case of "we thought there should be async stuff happening, now there's not but we can't break backwards compatibility"? It's how we ended up with WakeLockSentinel.release() returning a resolved promise after #299, for example.

I'm not saying I'm in favor of doing that, at this point I'm more curious about whether this is standard practice, or even if I'm looking at it from the wrong perspective.